app/javascript/components/teams/team-links.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <div class="flex flex-col w-full p-2 shadow sm:p-0 space-y-4 rounded-xl sm:shadow-none">
+    <div class="flex flex-row items-center justify-between ">
+      <span class="text-base font-medium">{{ $t('teams.links') }}</span>
+      <inline-svg
+        class="w-4 h-4 sm:hidden"
+        @click="expand"
+        :src="require(`../../../assets/images/icons/${icon}`)"
+      />
+    </div>
+    <div
+      v-if="expanded"
+      class="flex flex-col text-sm space-y-3"
+    >
+      <div
+        v-for="(link, index) in links"
+        :key="'link' + index"
+        class="flex flex-row rounded-lg sm:shadow sm:p-3 space-x-4"
+      >
+        <div class="flex flex-row space-x-3">
+          <div class="w-6 h-6 bg-black rounded-lg" />
+          <span>{{ link.platform }}</span>
+        </div>
+        <a
+          :href="link.url"
+          class="overflow-hidden text-blue-500 overflow-ellipsis"
+        >{{ link.url }}</a>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import resolveConfig from 'tailwindcss/resolveConfig';
+import tailwindConfig from '../../../../tailwind.config.js';
+
+const fullConfig = resolveConfig(tailwindConfig);
+const smBreakPoint = parseInt(fullConfig.theme.screens.sm.replace('px', ''), 10);
+
+export default {
+  data() {
+    return {
+      expanded: !(screen.width < smBreakPoint),
+      links: [
+        { platform: 'Notion', url: 'https://github.com/platanus' },
+        { platform: 'Github', url: 'https://trello.com/invite/b/HJrtFPIj/e1233f5daca0e122c7b972e6979ad1e3/golf' },
+      ],
+    };
+  },
+  mounted() {
+    window.addEventListener('resize', () => {
```

Comment:
Acá te faltó el `event` o no?

```suggestion
    window.addEventListener('resize', (event) => {
```

Me parece que te funciona porque aún está vigente [el `Window.event`](https://developer.mozilla.org/en-US/docs/Web/API/Window/event), que es global y es el evento del `listener` actual, pero puede estar deprecado en ciertos navegadores

Reply:
en efecto jaja